app/messages/_layout.email.html.erb

```diff
@@ -1,62 +1,116 @@
+            <td class="maincell" style="font-family:sans-serif; font-size:14px; vertical-align:top; box-sizing:border-box; padding:20px;">
 
                     <%= inner_html %>
 
-                  </td>
-                  <td width="36"></td>
-                </tr>
-                <tr>
-                  <td colspan="3" height="36"></td>
-                </tr>
-              </table>
             </td>
           </tr>
         </table>
-        <table cellpadding="0" cellspacing="0" align="center" border="0" style="background-color: #ffffff;">
-          <tr>
-            <td height="60"></td>
-          </tr>
-          <tr>
-            <td align="center"> <img src="<%= custom_logo || "https://lh3.googleusercontent.com/D1pgZeJZrBFc_n_EnEWpj9BQpVUaQ6xj2FYl1ZaxhDN4c-x8X69DqfJ258Eb0u7sXyuvYI533FkaCouBv6131tpxMbd1CpjZDEMr2kN4t8_Epitm77c" %>" alt=""></td>
-          </tr>
+        <!-- /.main -->
+
+        <!-- logo: branding -->
+        <table class="logo" style="width:100%; box-sizing:border-box; border-collapse:separate; mso-table-lspace:0pt; mso-table-rspace:0pt; ">
           <tr>
-            <td height="15"></td>
+            <td class="logocell" style="text-align:center; vertical-align:top; box-sizing:border-box; padding:10px;">
+              <img src="<%= custom_logo || "https://lh3.googleusercontent.com/D1pgZeJZrBFc_n_EnEWpj9BQpVUaQ6xj2FYl1ZaxhDN4c-x8X69DqfJ258Eb0u7sXyuvYI533FkaCouBv6131tpxMbd1CpjZDEMr2kN4t8_Epitm77c" %>" alt="">
+            </td>
           </tr>
+        </table>
+        <!-- /.logo -->        
+
+        <!-- footer: gray text below main -->
+        <table class="footer" style="width:100%; box-sizing:border-box; border-collapse:separate; mso-table-lspace:0pt; mso-table-rspace:0pt; ">
           <tr>
-            <td style="padding: 0; border-collapse: collapse;">
-              <table cellpadding="0" cellspacing="0" align="center" border="0">
-                <tr style="font-size: 11px; color: #a8b9c6; background-color: #ffffff; font-family: 'Open Sans', 'Lucida Grande', 'Segoe UI', Arial, Verdana, 'Lucida Sans Unicode', Tahoma, 'Sans Serif';" valign="top">
-                  <% if !content(:footer_link).blank? %>
-                    <td width="375" align="center">
-                    <%= content(:footer_link).html_safe %></td>
-                    <td width="50" align="center"> | </td>
-                  <% end %>
-                  <td width="375" align="center"><a href="<%= communication_profile_url %>">
-                    <%= t 'click_to_preferences', "Update your notification settings" %></a></td>
-                </tr>
-              </table>
+            <td class="footercell" style="font-family:sans-serif; font-size:14px; vertical-align:top; color:#999999; font-size:12px; text-align:center; padding:10px; box-sizing:border-box; ">
```

Comment:
This is really just the color of the "|" between the links, right?  Probably doesn't matter to much, but probably no reason to change now, since we ditched the gray background.